gums-core/src/main/java/gov/bnl/gums/account/AccountPoolMapper.java

```diff
@@ -163,6 +169,9 @@ public String getType() {
 
     @Override
     public AccountInfo mapUser(GridUser user, boolean createIfDoesNotExist) {
+        if (groupName != null && !groupName.equals("")) {
+            return new AccountInfo(getDB().retrieveMapping(user.getCertificateDN()), groupName);
```

Comment:
I'm not comfortable with this - why convert the grid credentials to a string here?  I'm not clear why this is necessary.

Why not simply add groupName to line 176 below?